spec/controllers/email_preferences_controller_spec.rb

```diff
@@ -2,14 +2,27 @@
 
 describe Users::EmailPreferencesController do
   describe "edit" do
-    let(:user) { create (:user) }
+    let(:user) { create :user, angular_ui_enabled: true }
 
-    it 'renders the beta interface when an unsubscribe token is present' do
+    it 'renders the beta interface for betea user when an unsubscribe token is present' do
```

Comment:
??

The logic's been simplified to say 'if the current_user_or_visitor has angular enabled, show the angular version, otherwise show the beta version'

This test is 'I am a user with angular off, and an unsubscribe token. I should get the beta interface' I don't think we can show beta users the angular interface for just one page, because there's no easy path back to beta from there.